### PR TITLE
feat: LC-498 lesson video

### PR DESCRIPTION
**Pull request by typeWolffo**
## Jira issue(s)
[LC-498](https://selleolabs.atlassian.net/browse/LC-498)

## Overview
- added video player as lesson view type

[LC-498]: https://selleolabs.atlassian.net/browse/LC-498?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/api/src/storage/migrations/0006_incerase_lesson_description_characters_count.sql
Comment (wielopolski) [inline apps/api/src/storage/migrations/0006_incerase_lesson_description_characters_count.sql:1]: Can you revert description in nice-data-seeds.ts:1493.
```
description:
              '<p>Artificial Intelligence (AI) refers to the simulation of human intelligence in machines programmed to think, learn, and make decisions. Below are some key concepts and terminologies essential to understanding AI:</p><ul><li><strong>Machine Learning (ML):</strong> A subset of AI focused on creating algorithms that allow computers to learn from and make predictions based on data. Example: A recommendation system suggesting movies based on your viewing history.</li><li><strong>Neural Networks:</strong> Inspired by the human brain, these are algorithms designed to recognize patterns and process data in layers, enabling tasks like image and speech recognition.</li><li><strong>Natural Language Processing (NLP):</strong> This involves teaching machines to understand, interpret, and generate human language. Example: Virtual assistants like Alexa or Siri.</li><li><strong>Computer Vision:</strong> A field of AI that enables computers to interpret and process visual data, such as images and videos. Example: Facial recognition technology.</li><li><strong>Deep Learning:</strong> A more complex subset of ML that uses large neural networks to analyze massive amounts of data and solve intricate problems, such as self-driving cars.</li><li><strong>Supervised vs. Unsupervised Learning:</strong><br>- <strong>Supervised Learning:</strong> The AI is trained on labeled data (e.g., images labeled as "cat" or "dog").<br>- <strong>Unsupervised Learning:</strong> The AI identifies patterns in unlabeled data without explicit instructions.</li><li><strong>Big Data:</strong> The large volume of structured and unstructured data generated by businesses and devices, which is essential for training AI models.</li><li><strong>Automation:</strong> AI is often used to automate repetitive tasks, freeing up human resources for more complex activities.</li><li><strong>Ethics in AI:</strong> As AI becomes more powerful, ensuring its ethical use (e.g., avoiding bias in decision-making) is critical for building trust.</li></ul><h3>Why These Concepts Matter</h3><p>Understanding these basic AI terms is the first step toward recognizing how AI can be applied in business. Each concept represents a building block of AI\'s potential to transform industries by increasing efficiency, improving decision-making, and creating innovative solutions.</p>',

**Review by wielopolski — APPROVED** [review]
LGTM 🍻

File: apps/api/src/storage/migrations/0006_incerase_lesson_description_characters_count.sql
Comment (typeWolffo) [reply apps/api/src/storage/migrations/0006_incerase_lesson_description_characters_count.sql:1]: done

File: apps/web/app/modules/Courses/NewCourseView/NewCourseView.page.tsx [hunk 1]
```diff
@@ -26,6 +26,8 @@ export default function NewCourseViewPage() {
   //   },
   // ];
 
+  console.log(course);
```
Comment (piotr-pajak) [inline apps/web/app/modules/Courses/NewCourseView/NewCourseView.page.tsx:29]: ✂️

**Review by piotr-pajak — APPROVED** [review]

**Review by mateuszszczecina — APPROVED** [review]

**Pull request merged**